workers/ohsome_quality_analyst/geodatabase/client.py

```diff
@@ -131,7 +131,7 @@ async def get_feature_ids(dataset: str, fid_field: str) -> List[Union[int, str]]
 
 
 # TODO Rewrite to work with geojson.Feature as input
-async def get_area_of_bpolys(bpolys: Dict):
+async def get_area_of_bpolys(bpolys: Union[Polygon, MultiPolygon]):
```

Comment:
Do you only allow Polygons here? You could use `bpolys: Geometry` instead.

Reply:
At least we expect only Polygon and MultiPolygon since OQT works on areas for now. Input features to OQT should have those types as geometry.
Also since we calculate the area only those two geometries are making sense.

Reply:
Is this documented somewhere? The whole “what can I pass to OQT” in which format should be documented somewhere. (GeoJSON with a Polygon or MultiPolygon, optionally within a Feature, optionally within a FeatureCollection).

Reply:
Currently it is not documented. It would be nice to document it before the release.

Reply:
https://github.com/GIScience/ohsome-quality-analyst/issues/78